### PR TITLE
add new company - ADEVA

### PR DESCRIPTION
**Pull request by Hanee-Solanki**
_Adeva is a global talent network that enables work without boundaries by connecting organizations with distributed tech talent across the world_

.
- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [ ] You are an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Hanee-Solanki** [comment]
Hi @dougaitken !

I have made the necessary changes now. 

Regarding the fourth point in the checklist template, I read on the company's website that they kind of act like a bridge between the companies and the employees. So I do not think that the company directly hires employees but instead provides recruiting services. Moreover, all the positions that are offered are fully remote.

**Comment by dougaitken** [comment]
@Hanee-Solanki So this is still angled as a freelancer type thing. This list is for hiring into a company.

I took a quick look and found this part of the site. Can you edit the readme to be about working for rather than freelancing for, please? https://adevait.com/careers/job-openings

**Comment by Hanee-Solanki** [comment]
Hi @dougaitken!

Thank you so much for reviewing my PR again and clearing this doubt.

I will certainly edit the readme.

Thanks again for your time!

**Comment by dougaitken** [comment]
No problem @Hanee-Solanki request a review from me when you're ready.

**Comment by Hanee-Solanki** [comment]
Hi @dougaitken

Sorry but I am not able to request a review from my end. It's strange because I do not see any option here.

Please see the attached image: 
image


Can you please guide me since this is my first time contributing to an open-source project?

**Comment by dougaitken** [comment]
All good, I should have stated to mention here rather than a specific review.

Do you have changes on your own repository copy? The current edit is still angled to freelancers rather than the link/edit I mentioned.

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/adeva.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+AdonisJS, AngularJS, Haskel, Java, JavaScript, Kafka, Laravel, Node.js, PHP, Python, ReactJS, Ruby on Rails, and VueJS  
+
+## Office locations
+
+United States 
+
+## How to apply
+
+Create your account at [https://app.adevait.com/apply](https://app.adevait.com/apply). 
+Submit your application with your basic personal information and work experience.
```
[the remark this answers is not in the document]
Comment (Hanee-Solanki) [reply company-profiles/adeva.md]: Hi @dougaitken !

I have updated the link now. Please review the final commit and let me know if anything needs to be changed.

Thank you.

File: company-profiles/adeva.md [hunk 2]
```diff
@@ -40,5 +40,5 @@ United States
 
 ## How to apply
 
-Create your account at [https://app.adevait.com/apply](https://app.adevait.com/apply). 
-Submit your application with your basic personal information and work experience.
+ob openings can be found at our [careers page](https://adevait.com/careers/job-openings). Submit your application with basic personal information and a resume.
```
Comment (KieranRobson) [inline company-profiles/adeva.md]: quick typo 

`ob` to `Job`

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Thanks @Hanee-Solanki

**Pull request merged**